lib/vehicle.rb

```diff
@@ -1,18 +1,23 @@
 require 'date'
+require 'pry'
 
 class Vehicle
+  attr_accessor :plate_type
   attr_reader :vin,
               :year,
               :make,
               :model,
-              :engine
+              :engine,
+              :registration_date
```

Comment:
Good call on using an `attr_accessor` for `:plate_type` since you needed your `Facility` class to be able to update that attribute. You could have also done the same thing for `registration_date`, and then you wouldn't have needed to make the `new_registration_date` method down below.